kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/decision/DecisionNavigatorPresenterTest.java

```diff
@@ -79,6 +79,20 @@ public void setup() {
                                                        translationService));
     }
 
+    @Test
+    public void testGetDiagram(){
```

Comment:
I think there's a format issue here:
```suggestion
    public void testGetDiagram() {
```